paasta_tools/list_marathon_service_instances.py

```diff
@@ -89,7 +90,7 @@ def get_desired_marathon_configs(soa_dir):
                 soa_dir=soa_dir,
             ).format_marathon_app_dict()
             marathon_configs[marathon_config['id'].lstrip('/')] = marathon_config
-        except (NoDeploymentsAvailable, NoDockerImageError):
+        except (NoDeploymentsAvailable, NoDockerImageError, NoSlavesAvailableError):
```

Comment:
It is ok for NoDeployments and NoDocker to be silent.

This should not be silent and needs to `_log`